### ISSUE TITLE
Health checks run only on conditioned data seems to contradict Section 4.2 of NIST SP 800-90B

### ISSUE THREAD

**Issue by rt-acumen** [usnistgov/90B-Shall-Statements#28]
ID#74, Section 4.3: In the notes it states:

"Caveat allowed when the health tests are run only on conditioned data."

Can CMVP please provide a reason for this particular exemption? Our understanding is that health checks are intended to be conducted on the noise source per Section 4.2. Heath checks run on conditioned data would not provide much meaningful value. These checks would not identify failures or a reduction of entropy output from the noise source.

**Comment by celic** [comment]
I agree this does not provide much benefit to the noise source. This is why it is only allowed as a caveat. The decision to allow this as a caveat was based on what we believe would be a hard-stop for entropy sources trying to be validated now. In some cases moving where the health tests are applied is difficult, so we decided to allow a path forward for such cases. We do not expect many devices to take advantage of this particular caveat. We can continue to discuss though.
